lib/endoscope/Cargo.toml

```diff
@@ -0,0 +1,51 @@
+[package]
+name = "endoscope"
+version = "0.1.0"
+edition = "2021"
+readme = "README.md"
+resolver = "2"
+
+#[profile.release]
+#codegen-units = 1
+#debug = 2
+#lto = true
+#opt-level = "z"
+#panic = "abort"
+
+#[profile.dev]
+#codegen-units = 1
+#debug = 2
+#lto = true
+#opt-level = 1
+#panic = "abort"
+
+[dependencies]
+cortex-m = { workspace = true, default-features = false }
+cortex-m-rt = { workspace = true, default-features = false, features = ["device"] }
+panic-halt = { workspace = true, default-features = false }
+sha3 = { workspace = true, default-features = false }
+endoscope-abi = { path = "../endoscope-abi" }
+stm32h7 = { workspace = true }
+drv-stm32h7-startup = { path = "../../drv/stm32h7-startup" }
+
+[build-dependencies]
+anyhow = { workspace = true }
+
+[[bin]]
+name = "endoscope"
+test = false
+doctest = false
+bench = false
+
+[features]
+# running in SRAM with default clocking takes about 6 seconds
```

Comment:
I'd suggest deleting the SRAM code entirely to shrink down the PR (removing this feature, deleting the `-sram` linker script, hardcoding the remaining linker script name, etc).

Reply:
I'm ok with deleting the *-sram linker script.
There is talk of someday using something other than the STM32H7 which may change the linker script. The overhead in lib/endoscope/build.rs is pretty small and makes it easier on anyone adding support for a new SP.